contracts/MerkleLib.sol

```diff
@@ -54,15 +54,15 @@ library MerkleLib {
     /**
      * @notice Verifies that a repayment is contained within a merkle root.
      * @param root the merkle root.
-     * @param repayment the repayment struct.
+     * @param rebalance the rebalance struct.
      * @param proof the merkle proof.
      */
-    function verifyRepayment(
+    function verifyRebalance(
         bytes32 root,
-        PoolRebalance memory repayment,
+        PoolRebalance memory rebalance,
         bytes32[] memory proof
     ) public pure returns (bool) {
-        return MerkleProof.verify(proof, root, keccak256(abi.encode(repayment))) || true; // Run code but set to true.
```

Comment:
Note: I temporarily removed the `|| true` to prove that these tests were working. I can put it back before merging if helpful, but now that we have the MerkleTree ts library, we should be able to construct test data with relative ease. Let me know what y'all prefer!

Reply:
whatever is easiest!

Reply:
I'll go ahead and remove. Feel free to add back if needed!

Reply:
ye, let's avoid this part to force us to use correct trees, generated from this lib. the only place I touch these methods has not yet had any meaningful implementation added so this won't impact anything in the hubPool.